Update 06-script.md

matiasandina opened the pull request:
Arreglo tildes y pequeñas cosas. Esta pregunta estaba metida dentro de la frase y no sé como traducirla --- "¿Cuándo se ejecutará como bash example.sh * .pdb si contuviera las siguientes líneas?"

Please delete the text below before submitting your contribution. 

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

matiasandina committed:
Update 06-script.md

Arreglo tildes y pequeñas cosas. Esta pregunta estaba metida dentro de la frase y no sé como traducirla --- "¿Cuándo se ejecutará como bash example.sh * .pdb si contuviera las siguientes líneas?"

orchid00 commented:
Hola @matiasandina ya lo ví, no es una traducción muy literal pero ahí va mi sugerencia:
En inglés: https://github.com/swcarpentry/shell-novice/blob/gh-pages/_episodes/06-script.md
Explain what a script called example.sh would do when run as `bash example.sh *.pdb` if it contained the following lines:

En español: si falta agregar los back ticks `` para que se haga el highlight de codigo

Con un script llamado `example.sh` explica cual sería el resultado de ejecutar
`bash example.sh *.pdb`
considerando que el script tenga una de estas tres opciones a la vez:

Alguien más quisiera comentar? @fmichonneau @ErinBecker @helysalgado ?

Helysalgado commented:
Propongo :

Explica, para los siguientes tres ejemplos, que hace el script llamado example.sh cuando lo ejecutas como  ``bash example.sh *.pdb``  : 

------
Me gusto @orchid00 que introdujeras que son tres ejemplos, de esta manera queda mucho más claro que decir  _que contiene las siguientes lineas_.

matiasandina committed:
Update 06-script.md

Agrego traduccion sugerida por @Helysalgado  "Explica, para los siguientes tres ejemplos, que hace el script llamado example.sh cuando lo ejecutas como bash example.sh *.pdb"

ErinBecker commented:
Gracias @matiasandina!
